src/GitLink/project.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "dependencies": {
+    "Catel.Core": "4.3.0",
+    "GitTools.Core": "1.0.0-unstable0021",
+    "ImpromptuInterface": "6.2.2",
+    "LibGit2Sharp": "0.21.0.176",
+    "ReadOnlySourceTree": {
+      "version": "0.1.37-beta",
+      "suppressParent": "none"
+    },
+    "Nerdbank.GitVersioning": {
```

Comment:
Before I simply remove, I'd like to see how GitVersion fills in all the requirements that NB.GV does. I expect we can switch back, but I'd like to switch without losing any functionality.